crates/sui-types/src/base_types.rs

```diff
@@ -33,9 +32,9 @@ use crate::error::ExecutionError;
 use crate::error::ExecutionErrorKind;
 use crate::error::SuiError;
 use crate::object::{Object, Owner};
-use crate::sui_serde::Base64;
 use crate::sui_serde::Hex;
 use crate::sui_serde::Readable;
+use crate::sui_serde::{Base64, Encoding};
```

Comment:
Thanks, crosschecked that we use the secure (non-malleable) base64ct crate.

Reply:
> Happy with this change. Wondering though if in a future PR base64 should move to `fastcrypto` where there we can control which backing lib is used. Ideally we need every crypto related aspect to be loaded from `fastcrypto` (and encoding schemes are part of it due to security implications, ie malleability)

makes sense, we can do that as a next step (after we finalize if we are replacing base64 to something else)